khelo/assets/locales/app_en.arb

```diff
@@ -303,6 +311,10 @@
   "match_detail_match_info_tab_title": "Info",
   "match_detail_highlight_tab_title": "Highlight",
   "match_detail_overs_tab_title": "Overs",
+  "match_detail_match_not_started_error_title": "Match has not started yet.",
+  "match_detail_error_description_text": "A detailed will be available after the start of the match.",
```

Comment:
`A detailed will be` -> this wording doesn't seems right, can we change this with `The details will be`